pyanaconda/modules/storage/partitioning/interactive_utils.py

```diff
@@ -403,3 +404,31 @@ def collect_file_system_types(device):
         supported_types.add(device.original_format.name)
 
     return list(supported_types)
+
+
+def collect_device_types(device, disks):
+    """Collect supported device types for the given device.
+
+    :param device: a device
+    :param disks: a list of selected disks
+    :return: a list of device types
+    """
+    # Collect the supported device types.
+    supported_types = set(SUPPORTED_DEVICE_TYPES)
+
+    # Include the type of the given device.
+    supported_types.add(devicefactory.get_device_type(device))
+
+    # Include md only if there are two or more disks.
+    if len(disks) > 1:
```

Comment:
if len(disks) <= 1: ? (given the type is discarded)

Reply:
Fixed in https://github.com/rhinstaller/anaconda/pull/2040/commits/77c9b8e2b996d5ec163dbd81b2c124b0a06039aa.